tests/components/dynalite/test_light.py

```diff
@@ -38,7 +44,38 @@
     assert device.async_turn_on.mock_calls[0] == call(aaa="bbb")
     await dyn_light.async_turn_off(ccc="ddd")
     assert device.async_turn_off.mock_calls[0] == call(ccc="ddd")
-    with patch.object(dyn_light, "hass"):
-        with patch.object(dyn_light, "schedule_update_ha_state") as update_ha:
-            dyn_light.try_schedule_ha()
-            update_ha.assert_called_once()
+
+
+async def test_supported_features():
+    """Test supported feaures didn't change."""
+    device = Mock()
+    bridge = Mock()
+    dyn_light = DynaliteLight(device, bridge)
```

Comment:
Don't create the entity directly. Set up the component and platform which will add the entity. Then assert the state attributes via the core state machine.

Reply:
fixed